src/components/AssetDetails/AssetVerifiedComponent.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Icon, Table } from 'semantic-ui-react';
+
+import DropdownComponent from '../common/DropdownComponent';
+
+class AssetVerifiedComponent extends Component {
+  state = {
+    show: false,
+    isChecked: this.props.assetDetail.verified,
+    optionText: this.props.assetDetail.verified ? 'Yes' : 'No'
+  };
+
+  handleDropdownChange = (event, data) => {
+    event.stopPropagation();
+
+    let dropdownValue = false;
+
+    if (data.value === 'Yes') {
```

Comment:
I don't think the `if...else` block will be necessary when you pass the value of the dropdown as either `true` or `false`. Have you tried it that way? That is:
```jsx
<DropdownComponent
  ...
  options={[{ text: 'Yes', value: true }, { text: 'No', value: false }]}
  placeholder="Select Verification Status"
   ...
 />

Reply:
Having true as a value gives a weird look to it. At one point the `yes` and `no` are no longer visible. And I feel a user might have a problem selecting the values in the sense that they might be afraid they have not applied an option. At this point I have removed the `if` as well as `optionText` within state.

![screen recording 2018-12-18 at 12 27 pm](https://user-images.githubusercontent.com/31322228/50144586-ad8a2900-02c0-11e9-92c9-bafb527f9978.gif)

Reply:
Alright...then leave the values as they are and just retrieve the option text value in the render function. Then you can have something like:
```jsx
handleDropdownChange = (event, data) => {
   event.stopPropagation();
// Note that this will be either true or false. Delete this comment after reading :)
   const dropdownValue = data.value === 'Yes';

   this.setState({
     isChecked: dropdownValue
   });
 };